src/Keyboard_hu_HU.h

```diff
@@ -0,0 +1,52 @@
+/*
+  Keyboard_hu_HU.h
+
+  Copyright (c) 2022, Edgar Bonet
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef KEYBOARD_HU_HU_h
+#define KEYBOARD_HU_HU_h
+
+#include "HID.h"
+
+#if !defined(_USING_HID)
+
+#warning "Using legacy HID core (non pluggable)"
+
+#else
+
+//================================================================================
+//================================================================================
+//  Keyboard
+
+// hu_HU keys
+#define KEY_O_SLASH             0x2e
+#define KEY_O_TWO_DOTS          0x27
+#define KEY_O_DOUBLE_SLASH      0x2f
+
+#define KEY_U_SLASH             0x30
+#define KEY_U_TWO_DOTS          0x2d
+#define KEY_U_DOUBLE_SLASH      0x31
+
+#define KEY_A_SLASH             0x34
+
+#define KEY_E_SLASH             0x33
+
+#define KEY_I_SLASH             0x32
```

Comment:
You should add 136 to the scan codes here, e.g.

```c++
#define KEY_O_SLASH             (136+0x2e)
```

This is for reasons pertaining to the internal encoding used by the library: codes starting at 136 are interpreted as (shifted) raw scan codes. In contrast, `0x2e` (the same as `'.'`) is interpreted as the ASCII full stop.

I suggest also renaming these macros:

* `KEY_*_SLASH` → `KEY_*_ACUTE`. This is in order to be consistent with the Unicode names of the characters (“LATIN SMALL/CAPITAL LETTER A/E/I/O/U WITH ACUTE”), and with the names already used in the library.

* `KEY_*_DOUBLE_SLASH` → `KEY_*_DOUBLE_ACUTE`, for consistency with Unicode (there is no double acute in the library yet).

* `KEY_*_TWO_DOTS` → `KEY_*_UMLAUT` (for consistency with the de\_DE and sv\_SE layouts in this library) or `KEY_*_DIARESIS` (for consistency with Unicode).

Reply:
I'm using copilot. It suggested ACUTE but I looked it up on google and I saw a boy with a hurt finger and I was like no thank you :D Looks like copilot was right after all...

Also thank you for typing out the fix so accurately.